modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -155,3 +156,36 @@ VM_NAME='fcos-node01'
 govc device.serial.add -vm "${VM_NAME}"
 govc device.serial.connect -vm "${VM_NAME}" "[datastore] ${VM_NAME}/console.log"
 ----
+
+== Modifying OVF metadata
+
+NOTE: While we provide these instructions for modifying the OVF metadata, we cannot
+guarantee that any modifications to the OVF metadata will result in a usable
+guest VM.
+
+Fedora CoreOS ships the VMware OVA with metadata defaults defined in the OVF
+file contained within the OVA file. The hardware version and default firmware
+are of particular interest, as they can affect how the guest VM operates in the
+hypervisor. These defaults may prevent the use of FCOS as a guest VM in older
+versions of VMware products and it may be necessary to modify the metadata in
+order to use FCOS as a guest VM.
+
+NOTE: The Fedora CoreOS project will not make any changes to the hardware
+version in the OVF metadata of the VMware OVA that would exclude non-EOL
+platforms from being supported.
+
+The VMware OVA is simply a tarball that contains the files `disk.vmdk` and
+`coreos.ovf`. In order to edit the metadata used by FCOS as a guest VM, you
+should untar the OVA artifact, edit the OVF file, then create a new OVA file.
+
+In the example below, the hardware version in the OVF is changed from the
+default value of `17` to an older version of `13`. (Note: the defaults in the
+OVF are subject to change.)
+
+[source, bash]
+----
+tar -xvf fedora-coreos-35.20220213.3.0-vmware.x86_64.ova
+sed -i 's|vmx-17|vmx-13|' coreos.ovf
+tar -H posix -cvf fedora-coreos-35.20220213.3.0-vmware-vmx-13.x86_64.ova coreos.ovf disk.vmdk
+----
```

Comment:
Let's avoid breaking user scripts when we change the default, and avoid hardcoding version numbers.

```suggestion
[source,bash,subs="attributes"]
----
tar -xvf fedora-coreos-{stable-version}-vmware.x86_64.ova
sed -iE 's/vmx-[0-9]*/vmx-13/' coreos.ovf
tar -H posix -cvf fedora-coreos-{stable-version}-vmware-vmx-13.x86_64.ova coreos.ovf disk.vmdk
----
```